Wabuluka/epic_mail
# Ch 164592359 setting up development environment 2

#### What does this PR do?
Refactored code for the Epicmail from procedure code to OOP
#### Description of Task to be completed?
In this PR, I have provided for user signup, login endpoints and validators. I have also provided for create message , delete message, get all, get one, which are to be modified to any requirements. 
#### How should this be manually tested?
Use Postman with the following details
```
endpoint for creating a message:
http://127.0.0.1:5000/epicmail/api/v2/messages
```

```
{
    "data": {
        "address": "webbiewabuluka@gmail.com",
        "createdby": "dwabuluka@gmail.com",
        "createdon": "Fri, 22 Mar 2019 17:36:38 GMT",
        "id": 1,
        "message": "I had taken long with out seeing you so i decided to message you to see if we can schedule a meet up",
        "status": "sent",
        "subject": "Hello world"
    },
    "message": "Message has been created successfully",
    "status": 201
}
````
#### What are the relevant pivotal tracker stories?
#### Screenshots (if appropriate)
![message_1](https://user-images.githubusercontent.com/17998079/54831579-a57cc380-4ccb-11e9-8c20-d55ef43ef903.png)


@kenneth051 @eubule @Busobozihakim
